gke/standard/zonal/secondary-boot-disk/main.tf

```diff
@@ -0,0 +1,44 @@
+/**
+* Copyright 2024 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+# [START gke_standard_secondary_boot_disk]
+resource "google_container_cluster" "cluster" {
+  name                = "default-cluster"
+  location            = "us-central1-a"
+  initial_node_count  = 1
+  deletion_protection = false
```

Comment:
nit: can we add a comment on why this is included similar to: https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/gke/standard/zonal/multi-zone/main.tf#L24

Reply:
Added!